pymobiledevice3/cli/webinspector.py

```diff
@@ -159,6 +166,48 @@ def jsshell(lockdown: LockdownClient, url):
         inspector.close()
 
 
+async def inspector_js_loop(inspector: WebinspectorService, app: Application, page: Page):
+    inspector_session = await inspector.inspector_session(app, page)
+    await inspector_session.runtime_enable()
+    while True:
+        exp = input('> ')
+        try:
+            print_json(await inspector_session.runtime_evaluate(exp))
+        except InspectorEvaluateError:
+            pass
+        except NotImplementedError:
+            pass
+
+
+@webinspector.command(cls=Command)
+@click.option('-t', '--timeout', default=3, show_default=True, type=float)
+def inspector_jsshell(lockdown: LockdownClient, timeout):
+    """
+    Opt in:
+
+        Settings -> Safari -> Advanced -> Web Inspector
+    """
+    inspector = WebinspectorService(lockdown=lockdown)
+    inspector.connect(timeout)
+    try:
+        safari_app = inspector.open_app(SAFARI)
+    except TimeoutError as e:
+        raise TimeoutError(f'Unable to launch application by bundle `{SAFARI}`') from e
```

Comment:
Don't raise the error again, just log it with error and exit

Reply:
done